app/src/main/java/ro/code4/monitorizarevot/fragment/BranchDetailsFragment.java

```diff
@@ -109,6 +112,12 @@ public void onClick(View v) {
                     Toast.makeText(getActivity(), R.string.invalid_branch_sex, Toast.LENGTH_SHORT).show();
                 } else if (timeEnter == null) {
                     Toast.makeText(getActivity(), R.string.invalid_branch_time_in, Toast.LENGTH_SHORT).show();
+                }
+                else if (timeLeave == null) {
```

Comment:
@afericean the leave time can be null. we do not need to check for that. we just have to make sure the user doesn't select a leaving time that is before the arrival time. could you please remove this check?